docs/build.md

```diff
@@ -0,0 +1,51 @@
+# Building the Kimai Docker
+
+The same docker file is used to build all the tagged images and is configured by combination of build arguments and 
+targets.
+
+## Targets
+
+The docker file has many staging targets but two functional builds are `prod` and `dev` which correspond to the 
```

Comment:
Whitespace after `"the "` violates markdownlint rule [MD009 - Trailing spaces](https://github.com/DavidAnson/markdownlint/blob/v0.20.4/doc/Rules.md#md009).